shavar/e2e-test/prefs.ini

```diff
@@ -55,8 +54,8 @@ browser.safebrowsing.provider.mozilla.lists = mozstd-track-digest256,mozstd-trac
 [mozstaging]
 urlclassifier.trackingTable = test-track-simple,mozstdstaging-track-digest256,mozfullstaging-track-digest256
 urlclassifier.trackingWhitelistTable = test-trackwhite-simple,mozstdstaging-trackwhite-digest256
-urlclassifier.disallow_completions = test-malware-simple,test-phish-simple,test-unwanted-simple,test-track-simple,test-trackwhite-simple,goog-downloadwhite-digest256,mozstd-track-digest256,mozstd-trackwhite-digest256,mozstdstaging-track-digest256,mozfullstaging-track-digest256,mozstdstaging-trackwhite-digest256
-browser.safebrowsing.provider.mozilla.lists = mozstd-track-digest256,mozstd-trackwhite-digest256,mozstdstaging-track-digest256,mozfullstaging-track-digest256,mozstdstaging-trackwhite-digest256
+urlclassifier.disallow_completions = test-malware-simple,test-phish-simple,test-unwanted-simple,test-track-simple,test-trackwhite-simple,goog-downloadwhite-digest256, mozstdstaging-track-digest256,mozfullstaging-track-digest256,mozstdstaging-trackwhite-digest256
```

Comment:
I would remove the stray space between `goog-downloadwhite-digest256` and `mozstdstaging-track-digest256` because I'm not sure we support that and may ignore that list.